test/sql/gapfill.sql

```diff
@@ -1,21 +0,0 @@
--- This file and its contents are licensed under the Apache License 2.0.
--- Please see the included NOTICE for copyright information and
--- LICENSE-APACHE for a copy of the license.
-
-\unset ECHO
-\o /dev/null
-\ir include/test_utils.sql
-\o
-\set ECHO queries
-\set VERBOSITY default
-
-\c :TEST_DBNAME :ROLE_SUPERUSER
-
-SELECT allow_downgrade_to_apache();
-SET timescaledb.license_key='ApacheOnly';
-
-\set ON_ERROR_STOP 0
-SELECT locf(1);
```

Comment:
Why don't we want this test anymore? Was it moved to another file?

I guess it tests that gapfill doesn't work under Apache license?

Reply:
We do, this moved to license test